outer.c

```diff
@@ -291,7 +292,8 @@ void outer_helper_spawn(struct outer_helper *helper)
 	/* Notify sibling that we're done persisting their proc files
 	   and/or changing their [ug]id map */
 	int ok = 1;
-	write(pipefds_in[1], &ok, sizeof (ok));
+	size_t count = write(pipefds_in[1], &ok, sizeof (ok));
+	assert(sizeof ok == count);
```

Comment:
micro-nit: I know parentheses are optional after the sizeof keyword, but I'd rather always have them